slides/ds101-worskhop.mdx

```diff
@@ -195,6 +195,49 @@ to find out the relevant parts of your design system.
 
 ---
 
+# Pattern Journey
+
+<Notes>
+
+# Why
+
+We are doing this exercise to outline the process of maintaining a design system. As a result,
+we get to know what is happening, who are the actors on the way and what tools they are
+using.
+
+We share stories about how this exercise was run on real projects and how to interpret its
+outcomes.
+
+# How
+
+Note! When running the exercise with a group of unfamiliar people, it is easy to go with
+a specific pattern. Say, login form. But when repeating this exercise again with their
+project teams, people can outline the whole process.
+
+Participants go to the whiteboard with prepared canvas. The exercise activities:
+
+1. Introducing canvas
+1. Naming the 5 stages (15 minutes)
+1. "What" lane (45 minutes)
+1. "Who" lane (15 minutes)
+1. "How" lane (15 minutes)
+
+# Conclusion
+
+Every time when we run this exercise, we get very different results. But interpretation of
+these results was always fruitful. In this case, we analyse the outcome we get now and
+share information about pattern journeys at our projects.
```

Comment:
Can you add slides showing how it was in your projects showing boards and going through interpretation?

Reply:
If you mean the workshop board itself, its NDA :-( The Pattern Journey is public, I was thinking to add slides about it and about your state machine in my next PR.